charts/consul/templates/connect-inject-deployment.yaml

```diff
@@ -193,6 +193,26 @@ spec:
                 -consul-sidecar-cpu-request={{ $consulSidecarResources.requests.cpu }} \
                 {{- end }}
                 {{- end }}
+          livenessProbe:
+            httpGet:
+              path: /readyz/ready
+              port: 9445
+              scheme: HTTP
+            failureThreshold: 2
+            initialDelaySeconds: 1
+            periodSeconds: 2
+            successThreshold: 1
+            timeoutSeconds: 5
+          readinessProbe:
+            httpGet:
+              path: /readyz/ready
+              port: 9445
+              scheme: HTTP
+            failureThreshold: 2
+            initialDelaySeconds: 2
```

Comment:
This is more out of curiosity than anything else, but is there a reason the `initialDelaySeconds` is 2 here while it's value is 1 in the livenessProbe?

Reply:
We've used the same configuration as before for both probes and didn't re-evaluate whether those make sense since it's essentially the same check as before. We'll take a deeper look on Monday.

Reply:
We took another look, the two probes are initially triggered 1 second apart to keep the two requests staggered.

Reply:
AHA!! good find!! thanks for the update Nicoleta!!